uptime-kuma/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
     stop_grace_period: 1m
     volumes:
       - ${APP_DATA_DIR}/data/uptime-kuma:/app/data
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
We can't allow an app to have access to the Docker unix socket for security reasons, as they could take over and manipulate any running container on the host.

As an alternative, you can have Uptime Kuma ping the hostname `lightning_lnd_1` for the LND container to check if it's running or not.